src/pug/blog/gyro_ratcheting.pug

```diff
@@ -0,0 +1,162 @@
+extends _blog.pug
+
+block append subtitle
+  | Gyro ratcheting and flick stick
+
+block content
+  h2 Gyro ratcheting and flick stick
+  hr
+  p: i 28 February 2023
+  hr
+
+  h3 Introduction and disclaimer
+  p This blog entry is an answer to the very #[em frequently asked questions] about using flick stick on the Alpakka. We wrote this as a consolidation of many things we learnt while researching gyro controls, and it is a mix of data, rationales, and thoughts.
+  p This is not meant to be a confrontational write-up, that's why it is titled #[em ratcheting AND flick stick] and not #[em ratcheting VS flick stick].
+  hr
+
+  h3 Early days of gyro as aiming method
+  p Using angular rate sensors (aka gyroscope, or simply gyro) as a method for camera control in videogames is relatively new. Some commercial controllers included an IMU (Inertial Measurement Unit) sensor, but it was never intended as primary input method for camera control, but mostly to enable new alternative "experiences", such as for example, playing golf swinging the controller.
+  p It was up to gyro enthusiasts to start trying new ways to use the gyro on their controllers, sometimes by using custom mapping done with 3rd party software on PC, or even by coding custom firmware for their proprietary controllers.
+  p The first approaches, #[em first generation] of gyro aiming , were as simple as using the secondary thumbstick as a standard camera control (relative turn rate) while employing the gyro for fine camera control (for example adjusting the last milli-degrees necessary for landing a headshot in a shooter). These IMU sensors were not very accurate, so they had to be configured with a very low sensitivity, so big movements translated into small in-game camera turns.
+  p Then later, the #[em second generation] of gyro aiming evolved into two parallel solutions: #[em ratcheting] and #[em flick stick], these approaches the lack of gyro precision in different ways:
+  table.manual.col3
+    tr
+      th
+      th Ratcheting<br>(on generic controllers)
+      th Flick stick
+    tr
+      th Full scale
+      td Gyro tuned for large turns
+      td
+        div Thumbstick perimeter mapping
+        div: em [workaround]
+    tr
+      th Fine control
+      td
+        div Gyro with lot of filtering
+        div (smoothing and acceleration)
+        div: em [workaround]
+      td Gyro tuned for small turns
+  hr
+
+  h3 Gyro ratcheting
+  p Ratcheting is when gyro is used as unique method of control, so the thumbstick is not used for camera control anymore. This approach requires a solution to engage and disengage the gyro, for re-centering purposes, similar to re-centering a mouse (the mouse can be lifted and re-centered while "disengaged").
```

Comment:
Ratcheting refers to a usage situation, where the gyro is used as the only method of camera control.